e2e/cypress/integration/monthly-staffing.js

```diff
@@ -47,18 +52,20 @@ describe('Monthly Staffing', function () {
   describe('When adding staff using the monthly staff view', function () {
 
     let cellToTest = ".htCore tbody :nth-child(1) :nth-child(2)";
-    xit("If I enter staff for the current month those staff should still be visible if I change months and change back", function () {
+    it("If I enter staff for the current month those staff should still be visible if I change months and change back", function () {
       saveShifts();
-
       setRoles(["staff:edit"]);
 
       cy.visit('/v2/test/live#terminal/T1/staffing/15///');
+      waitAndHardReload();
```

Comment:
I'm not sure adding these wait & reloads in before checking each result on the page is really making the test valid. We want the app to update what's on the page without having to reload it. So although this validates that the data in the app changes, it doesn't validate that the display correctly updates without manual intervention

Reply:
Precisely.
There is another JIRA ticket to fix the app and undo this PR.
https://jira.digital.homeoffice.gov.uk/browse/DRT-5206